ui/test/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    "extends": [
+        "../.eslintrc.js"
+    ],
+    "rules": {
+        "@typescript-eslint/no-explicit-any": "off"
```

Comment:
Do we want turn this off for all files?

Reply:
I only turned it off for files in the "test" folder. I prefer to have it on for other files